### ISSUE TITLE
Word Api 1.1/1.2 support 

### ISSUE THREAD

**Issue by emreozen** [WillemSleegers/tidystats-Word-add-in#17]
Would this add-in manifest show up on the Ribbon for earlier Word 2016  versions? Word Api 1.1 & 1.2?

It shows up with Word Api 1.3+ but does not show up on the ribbon for earlier ones:
![image](https://user-images.githubusercontent.com/9028038/118173353-7372d200-b3fb-11eb-8697-6d49cc3f9692.png)

**Comment by WillemSleegers** [comment]
Hm.. how are you setting the Word API version? I have set the minimum version requirement in the Netlify manifest to 1.1 so it should work.

**Comment by WillemSleegers** [comment]
I recommend installing it from the Add-in store by the way, rather than side loading one of the manifest files. Is there a reason you're using the Netlify manifest?

**Comment by emreozen** [comment]
> I recommend installing it from the Add-in store by the way, rather than side loading one of the manifest files. Is there a reason you're using the Netlify manifest?

I am using Word version 16.0.4266.1003 which supports API 1.1 and 1.2 but not 1.3. 

I just removed the sideloading and downloaded from the store, but I am having the same issue. Basically the app logo does not show up in the Home ribbon. If I download it in my newer computer with Office365, it shows up on the Home tab.

**Comment by WillemSleegers** [comment]
Hm.. it should show up in the Insert tab, but I guess you would have found it there. 

I'll look into it and see if I can figure this out because I shouldn't be requiring 1.3.

**Comment by emreozen** [comment]
> Hm.. it should show up in the Insert tab, but I guess you would have found it there.
> 
> I'll look into it and see if I can figure this out because I shouldn't be requiring 1.3.

thanks - I have the software package here: https://drive.google.com/file/d/18Wztc5gpYTFYE6R6qkNrJNsCRZ17Wi2h/view?usp=sharing

**Comment by WillemSleegers** [comment]
So, I installed the version of Word that you were using and I also do not see the add-in show up in the Insert tab. I do see that the add-in works, because if you Add the add-in from the Add-in Store, it opens the add-in and it's functional. 

I then updated the version of Word to the latest version (via File -> Account -> Office Updates) and now the add-in is visible in the Insert pane. The version I'm running now is 16.0.12527.21912. Is it possible for you to update your version of Word? This doesn't have to be the latest version I think, but simply checking whether there are new updates available might be the solution.
